numpy/array_api/_array_object.py

```diff
@@ -995,7 +1009,9 @@ def __rxor__(self: Array, other: Union[int, bool, Array], /) -> Array:
         res = self._array.__rxor__(other._array)
         return self.__class__._new(res)
 
-    def to_device(self: Array, device: Device, /) -> Array:
+    def to_device(self: Array, device: Device, /, stream: Optional[Union[int, Any]] = None) -> Array:
```

Comment:
```suggestion
    def to_device(self: Array, device: Device, /, stream: None = None) -> Array:
```
Considering numpy's array-api implementation only accepts `None` as stream-value.

Reply:
It would be nice if NumPy could keep the signatures as prescribed in the spec, otherwise we will have to fix it downstream... It's simple enough to reject any non-None objects here, right?

Reply:
Can you elaborate why this would not be up to spec? The `__dlpack__` [docs](https://data-apis.org/array-api/latest/API_specification/array_object.html#dlpack-self-stream-none) explicitly state that devices lacking stream-support should only accept `None` for the `stream` parameter.

Reply:
For NumPy it's ok. For CuPy we will have to fix it when we apply the patch from this PR. It's an extra step that I have to note to remind myself 😥